proxy/src/main/java/ru/leymooo/botfilter/config/Settings.java

```diff
@@ -49,7 +49,7 @@ public class Settings extends Config
     @Comment("Проверять ли на бота при заходе на сервер во время бот атаки, не зависимо проходил ли проверку или нет")
     public boolean FORCE_CHECK_ON_ATTACK = true;
     @Comment("Показывать ли онлайн с фильтра")
-    public boolean SHOW_ONLINE = true;
+    public boolean SHOW_ONLINE = false;
```

Comment:
```suggestion
    public boolean SHOW_ONLINE = true;
```
Please do this in another Pull-Request as it is something complete different.